admin/upgrade.php

```diff
@@ -28,8 +28,8 @@
 
 	// From what are we upgrading?
 	if ( isset( $_GET['oldver'] ) && isset( $_GET['oldsql'] ) ) {
-		$oldver = yourls_sanitize_version( $_GET['oldver'] );
-		$oldsql = yourls_sanitize_version( $_GET['oldsql'] );
+		$oldver = (string)( $_GET['oldver'] );
+		$oldsql = (string)( $_GET['oldsql'] );
```

Comment:
Why not using the sanitize function? And not really related to the encoding issue, right?

Reply:
Not related to encoding. It was breaking the build because the function doesn't do as advertised. I should take care of it btw.

Reply:
In fact I was using `yourls_sanitize_version()` for YOURLS version, like `1.5.1`, fine, but also for the SQL version, eg `480`, in this case in was not fine (not a "version number" and was returning `''`). Anyway type casting is enough here.